packages/matter.js/package.json

```diff
@@ -53,15 +53,7 @@
         "@matter.js/node": "*"
     },
     "devDependencies": {
-        "@matter.js/tools": "*",
-        "@matter.js/codegen": "*",
```

Comment:
This dependency was legit, src/build.config.ts uses `generateSubmoduleProxy` from codegen

Reply:
yes found that out too now ;) re-adding